### PR TITLE
a potential bug in Layer.

### PR DESCRIPTION
**Pull request by zcgit**
https://github.com/cocos2d/cocos2d-x/issues/2948

### CODE REVIEW

File: cocos2dx/layers_scenes_transitions_nodes/CCLayer.cpp [hunk 1]
```diff
@@ -77,7 +77,8 @@ bool Layer::init()
         Director * pDirector;
         CC_BREAK_IF(!(pDirector = Director::sharedDirector()));
         this->setContentSize(pDirector->getWinSize());
-        _touchEnabled = false;
+        //_touchEnabled = false;
```
Comment (dumganhar) [inline cocos2dx/layers_scenes_transitions_nodes/CCLayer.cpp]: I think delete the comment will be better.

File: cocos2dx/layers_scenes_transitions_nodes/CCLayer.cpp [hunk 2]
```diff
@@ -77,7 +77,8 @@ bool Layer::init()
...
+        setTouchEnabled(false);
         _accelerometerEnabled = false;
```
Comment (dumganhar) [inline cocos2dx/layers_scenes_transitions_nodes/CCLayer.cpp]: Does `_accelerometerEnabled = false` also need to be `setAccelerometerEnabled(false)`?

**Comment by zcgit** [comment]
oh , no , I'm sorry , I found a bug in CCScrollView after I modified it , I' thinking how to solve it ...

**Comment by dumganhar** [comment]
Thanks.

**Pull request merged**